src/io/flutter/view/FlutterView.java

```diff
@@ -110,17 +114,22 @@ public void actionPerformed(AnActionEvent event) {
     // manually opens the window when there is not yet a running app.
   }
 
-  private DefaultActionGroup createToolbar(@NotNull ToolWindow toolWindow, @NotNull FlutterApp app) {
+  private DefaultActionGroup createToolbar(@NotNull ToolWindow toolWindow, @NotNull FlutterApp app, Disposable parentDisposable) {
     final DefaultActionGroup toolbarGroup = new DefaultActionGroup();
     toolbarGroup.add(registerAction(new ToggleInspectModeAction(app)));
     toolbarGroup.addSeparator();
     toolbarGroup.add(registerAction(new DebugDrawAction(app)));
     toolbarGroup.add(registerAction(new TogglePlatformAction(app)));
     toolbarGroup.add(registerAction(new PerformanceOverlayAction(app)));
     toolbarGroup.addSeparator();
+    // TODO(pq): incorporate these into the heap display (maybe into a single combo selection).
     toolbarGroup.add(registerAction(new OpenTimelineViewAction(app)));
     toolbarGroup.add(registerAction(new OpenObservatoryAction(app)));
-    toolbarGroup.addSeparator();
+    // TODO(pq): push this into preferences or a menu option.
+    if (PerfService.DISPLAY_HEAP_USE) {
```

Comment:
+1 to using the gear menu (and defaulting to on) in the future (and top-level pref would work as well)